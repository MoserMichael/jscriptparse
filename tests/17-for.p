
println("loop over rangeof numbers")
for x range(1,10) {
    println(x)
}

println("loop over dictionary entries")

js='{ "name":"kermit", "surname":"frog", "profession":"tv producer" }'
dct = parseJsonString(js)
for k,v dct {
    println("key: {k} value: {v}")
}

println("loop over list entries")
for k [3,2,1] 
 println(k)

