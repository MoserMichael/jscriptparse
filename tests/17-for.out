loop over rangeof numbers
1
2
3
4
5
6
7
8
9
loop over dictionary entries
key: name value: kermit
key: surname value: frog
key: profession value: tv producer
loop over list entries
3
2
1
